keps/sig-node/1287-in-place-update-pod-resources/README.md

```diff
@@ -254,6 +277,9 @@ If a pod's RestartPolicy is `Never`, the ResizePolicy fields must be set to
 in the container being stopped *and not restarted*, if the system can not
 perform the resize in place.
 
+The `ResizePolicy` field is **mutable**, but must have an entry for every resizable resource type
```

Comment:
Is there a use case for this? Typically the pod owners (app owners) know in advance whether their app can be resized without restarts or not. That characteristic does not typically change during the lifetime of the app (pod). Unless there is a clear use case, this would add complexity to the implementation.

Reply:
The use case is for adding new resources to requests & limits. For example, if I have a pod with a CPU request & limit, and I want to add a memory limit to it. Today, that causes the API defaulting to add a new ResizePolicy for memory, but then the request fails validation because it's mutating an immutable field.

After some offline design discussions, we decided to change this to be append-only. In other words, the policy will still be mostly immutable, but the case I described above is allowed.

I'll update the KEP with this change, and make the usecase explicit here.

Reply:
SG